### DISCUSSION TITLE
Tilt support for styluses?

### DISCUSSION THREAD

**Discussion by beeirl** [fabricjs/fabric.js#7594] [Q&A]
Is there currently any support for (or are there any plans to add) tilt from pens (e.g. Apple pencil, S Pen etc)?

**Comment by asturur** [comment]
so i think tilt information can be used using the pointer event api.
We actually do suppor the pointer event, it would be a matter of exposing the property from the event.
What kind of support were you looking for? in what functionality?

**Reply by beeirl** [reply]
Thanks for getting back - I see what you are saying.
I would like to somewhat mimic the behaviour of the iOS notes app on the iPad where the pen switches into "highlight mode" when tilt is activated (it's also pressure-sensitive). Hope that makes sense.

**Reply by asturur** [reply]
well not much for me that i don't have an ipad/ios.
Can you enable the pointer events in fabricJS and put a console.log in the  mousedown/mousemove event of the pencil brush to see what is going on there on the event?
